### ISSUE TITLE
Black screen in Forza Horizon 4

### ISSUE THREAD

**Issue by IvoAndonov** [HansKristian-Work/vkd3d-proton#665]
I'm trying to play Forza Horizon using vkd3d but all I get is either fully black screen or the thing from the below screenshot. There is only sound and this output in terminal:

`300:fixme:d3d12_device_caps_init_feature_options1: TotalLaneCount = 1536, may be inaccurate.
300:fixme:d3d12_command_queue_init: Ignoring priority 0x64.
err:   DxgiFactory::CreateSwapChainForComposition: Not implemented
560:fixme:d3d12_root_signature_init_global: Ignoring root signature flags 0xd.
560:fixme:d3d12_root_signature_init_global: Ignoring root signature flags 0x1d.
588:fixme:rs_desc_from_d3d12: Ignoring MultisampleEnable 0x1.
560:fixme:vkd3d_dxbc_compiler_emit_resource_declaration: Unhandled instruction flags 0x100.
560:fixme:vkd3d_dxbc_compiler_emit_resource_declaration: Unhandled instruction flags 0x100.
560:fixme:d3d12_swapchain_init: Ignoring swap effect 0x4.
560:fixme:d3d12_swapchain_init: Ignoring swapchain flags 0x840.
warn:  DXGI: MakeWindowAssociation: Ignoring flags
560:fixme:d3d12_swapchain_GetFrameStatistics: iface 000000003f3a0050, stats 000000000c213178 stub!`

![ZgpxspX](https://user-images.githubusercontent.com/41172201/118471025-f6ec3600-b70f-11eb-8619-14f681b03d70.png)

**Comment by Blisto91** [comment]
@rado84-github Sorry for necro ping in old issue, but is this still an issue for you?

**Comment by rado84-github** [comment]
> @rado84-github Sorry for necro ping in old issue, but is this still an issue for you?

No. I purchased the game on Steam and it seems to be working somewhat fine with Proton Experimental. There are still some other issues but at least the black screen is gone.

**Comment by Blisto91** [comment]
Hot damn that was a fast answer.
Are the other issues vkd3d-proton related?

**Comment by rado84-github** [comment]
I'm currently viewing my inbox and an e-mail arrives instantly when there's a new reply, hence the fast answer. :) 
IDK, honestly. Trying to play multiplayer race often causes CTD, using the wheelspin also freezes the game, driving fast (350 kph or faster) causes textures to disappear and shortly after that - a CTD. Whether this is a vkd3d-proton problem or not - I have no idea, I'm not a programmer.

**Comment by Blisto91** [comment]
Lovely gameplay experience 😁

**Comment by mbriar** [comment]
Is this one nvidia? Crashes and freezes with FH 4 and 5 on nvidia I see reported all the time.

**Comment by rado84-github** [comment]
It is nvidia. But FH5 works fine mostly. On Forza 5 the only limitation is not to drive at speeds higher than 400 kph, otherwise the textures can't load that fast and the car appears to be riding the air, much like a Valkyrie. Otherwise everything else works perfectly fine, so if you do offroading, you won't see any difference between performances on Linux and Windows.
